README.md

```diff
@@ -20,34 +20,65 @@ yarn manypkg check
 yarn manypkg fix
 ```
 
-### Script Runner
+## Dictionary
 
-> Note: The following is not implemented currently
+- **private package** - A package that has `private: true`/is not published. It does not refer to a package published to a private registry here.
+- **internal package/workspace** - A package that is local/in the repo
+- **external package** - A package that is from a registry like npm
+- **range** - A [node-semver range](https://github.com/npm/node-semver#ranges)
 
-### In the root of a multi-package repo
+## Checks
 
-```
-yarn m <pkgName> add <depDescriptor>
-```
+## External mismatch
 
-```
-yarn m <pkgName> <scriptName>
-```
+The ranges for all dependencies(excluding `peerDependencies`) on external packages should be equal(`===`). This is so that only a single version of an external package will be installed because having multiple versions of the same package can cause confusion and bundle size problems especially with libraries like React that require there to only be a single copy of the library. It's important to note that this check does not enforce that only a single version of an external package is installed, only that two versions of an external package will never be installed because they're specified as dependencies of internal packages.
 
-```
-yarn m <binaryFromNodeModulesName>
-```
+There are vague thoughts that it should be possible to partially disable external mismatch rules but we have not yet had a use case for it so we have not addressed it yet.
 
-### In a package
+### How it's fixed
 
-```
-yarn m add <depDescriptor>
-```
+The highest range of the dependency is found as defined by [sembear](https://github.com/mitchellhamilton/sembear) and set as the range at every non-peer dependency place it is depended on.
```

Comment:
I'd love if we could point to a dictionary entry as opposed to just 'as per this exeternal package' - particularly because nobody will instantly know about sembear (yet).